docs/guides/docs/installation/index.mdx

```diff
@@ -17,13 +17,17 @@ This guide covers the following topics:
 
 ## Installing Rust
 

```

Comment:
We should add Playwrite tests to this if we're starting to use this across all the docs. If not in this PR we should open an issue for this ASAP

Reply:
Find the [issue here](https://github.com/FuelLabs/docs-hub/issues/81)